dev-requirements.txt

```diff
@@ -7,4 +7,5 @@ nose==1.3.7
 nose-cov==1.6
 nose-timer==0.7.1
 flake8==3.5.0
-
+tox==3.0.0
+tox-pyenv==1.1.0
```

Comment:
Not completely related, but can you please sort the entries in this file alphabetically?